lib/backend/k8s/resources/node.go

```diff
@@ -60,7 +60,13 @@ func (c *nodeClient) Create(ctx context.Context, kvp *model.KVPair) (*model.KVPa
 }
 
 func (c *nodeClient) Update(ctx context.Context, kvp *model.KVPair) (*model.KVPair, error) {
-	// Get a current copy of the node to fill in fields we don't track.
+	log.Debug("Received Update request on Node type")
+	// We switched to using UpdateStatus here to avoid conflicts in write
+	// https://github.com/kubernetes/kubernetes/issues/39113
+	return c.UpdateStatus(ctx, kvp)
+}
+
+func (c *nodeClient) UpdateStatus(ctx context.Context, kvp *model.KVPair) (*model.KVPair, error) {
```

Comment:
I don't think we need a separate function here for `UpdateStatus` - especially not a public one.

Let's just keep this as one function - `Update()` and simply change the API call we're making to k8s.